packages/agent-toolkit/src/core/tools/monday-apps-tools/consts/routes.consts.ts

```diff
@@ -57,21 +44,10 @@ export const API_ENDPOINTS = {
   },
 
   CODE: {
-    APP_VERSION_BASE: (appVersionId: number) => `${CODE_URL}/${appVersionId}`,
     GET_DEPLOYMENT_STATUS: (appVersionId: number) => `${CODE_URL}/${appVersionId}/deployments`,
     GET_DEPLOYMENT_SIGNED_URL: (appVersionId: number) => `${CODE_URL}/${appVersionId}/deployments/signed-url`,
-    UPLOAD_DEPLOYMENT: (appVersionId: number) => `${CODE_URL}/${appVersionId}/deployments`,
-    GET_LOGS: (appVersionId: number, logsType: string, additionalParams?: string) =>
```

Comment:
Why remove? Might be used to help debug when working on code

Reply:
it's not used anywhere